master/docs/relnotes/index.rst

```diff
@@ -10,6 +10,48 @@ Release Notes
 
 .. towncrier release notes start
 
+Buildbot ``0.9.6-35-gfb26e170d`` ( ``2017-04-27`` )
```

Comment:
Arghh.

It keeps adding to the commit index (I don't do commit -a or else). Do you know why ?
Surely because I run `make -C master/docs SPHINXOPTS=-W spelling` to compile docs

Does another command exists ?